### PR TITLE
DOCS-1480: Adds redirects for bad links from tigera.io

### PR DESCRIPTION
**Pull request by ctauchen**
Product Version(s):

N/A

Issue:

https://tigera.atlassian.net/browse/DOCS-1480

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

**Comment by ctauchen** [comment]
@girishshankaran Can you take a look?

File: static/_redirects [hunk 1]
```diff
@@ -170,7 +174,23 @@
...
 /security/tutorials/kubernetes-policy-demo/kubernetes-demo https://docs.tigera.io/calico-enterprise/latest/network-policy/get-started/kubernetes-demo 301
 /security/tutorials/protect-hosts https://docs.tigera.io/calico-enterprise/latest/network-policy/hosts/protect-hosts 301
+/master/security/simple-policy-cnx /calico-enterprise/latest/network-policy/ 301
+/security/adopt-zero-trust /calico/latest/network-policy/adopt-zero-trust 301
+/security/compliance-managed-cloud /calico-enterprise/latest/compliance/ 301
+/security/compliance-reports/ /calico-enterprise/latest/compliance/ 301
+/security/compliance-reports/overview /calico-enterprise/latest/compliance/ 301
+/security/kubernetes-policy /calico-enterprise/latest/network-policy/get-started/kubernetes-network-policy 301
+/security/logs/ /calico-enterprise/latest/visibility/elastic/overview 301
+/security/logs/ /calico-enterprise/latest/visibility/elastic/overview 301
```
Comment (girishshankaran) [inline static/_redirects]: Duplicate entry

**Review by girishshankaran — APPROVED** [review]
LGTM

File: static/_redirects (hunk 1 above)
Comment (ctauchen) [reply static/_redirects]: Thanks. Removed.

**Pull request merged**